server/util/jurisdiction_bulk_update.py

```diff
@@ -30,6 +33,26 @@ def process():
 
     process_file(session, file, process)
 
+    # If standardized contests file already uploaded, try reprocessing the
+    # standardized contests file as well, since it depends on jurisdiction names.
+    if election.standardized_contests_file:
+        logger.info(
+            f"START_REPROCESSING_STANDARDIZED_CONTESTS {dict(election_id=election.id)}"
+        )
+        # First, clear out the previously processed data.
+        election.standardized_contests = None
+        election.contests = []
+        election.standardized_contests_file.processing_started_at = None
+        election.standardized_contests_file.processing_completed_at = None
+        election.standardized_contests_file.processing_error = None
+        session.flush()  # Make sure process_file can read the changes we just made
```

Comment:
I think what you're doing here is updating the DB without committing... and we're sure `process_file` will be within the same transaction, right? If so, sounds good.

Reply:
Yes, the idea is to make sure those changes are readable by queries that happen in the same transaction in process_file. So instead of committing them (which would end the transaction and start a new one), we flush them, which moves them from the Sqlalchemy session to the database.